introduction/A_overview.md

```diff
@@ -1,8 +1,8 @@
 Phoenix is a web development framework written in Elixir which implements the server-side MVC pattern. Many of its components and concepts will seem familiar to those of us with experience in other web frameworks like Ruby on Rails or Python's Django.
 
-Phoenix provides the best of both worlds, high developer productivity _and_ high application performance. It also has some interesting new twists like channels for implementing realtime features and pre-compiled templates for blazing speed.
+Phoenix provides the best of both worlds: high developer productivity _and_ high application performance. It also has some interesting new twists like channels for implementing realtime features and pre-compiled templates for blazing speed.
```

Comment:
I agree that the comma is wrong here, but I'm not convinced that a colon is correct either.

What would you think about a dash? ". . . best of both worlds - high developer productivity  . . . "